src/common/util/game_settings.hpp

```diff
@@ -71,8 +71,8 @@ const std::chrono::milliseconds SERVER_BROADCAST_GAME_STATE_SLEEP(16);
 const std::chrono::milliseconds CLIENT_UNRELIABLE_RECV_SLEEP(SERVER_BROADCAST_GAME_STATE_SLEEP / 2);
 
 /* Networking */
-static const char* CLIENT_IP = "127.0.0.1";
+static const char* CLIENT_IP = "73.15.213.77";
```

Comment:
I dont want to commit this, so Im going to implement dynamically learning the client IP and using it before merging